polyfill/lib/duration.mjs

```diff
@@ -138,6 +138,12 @@ export class Duration {
   static from(...args) {
     return ES.CastDuration(...args);
   }
+  static fromNanoseconds(arg) {
+    return new this(0, 0, 0, 0, 0, 0, 0, 0, arg);
```

Comment:
Do we want to check this is a bigint first? I feel like we want to use `ToBigInt`, but I'm not sure how to do that in JS.

Reply:
Passing a BigInt here would be wrong.

Also fromNanoseconds() on duration sounds bad on many levels!

Reply:
To be clear, this was based on the code in `casts.mjs` above that got merged after the discussion at https://github.com/tc39/proposal-temporal/pull/163#discussion_r333414071. If you prefer removing the code from `casts.mjs` without adding these new static methods, that's fine by me as well.